content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -1,199 +1,96 @@
 # Sharing Experiments
 
-There are two types of remotes that can store experiments. Git remotes are
-distributed copies of the Git repository, for example on GitHub or GitLab.
+Regular <abbr>DVC repository</abbr> versions are typically synchronized among
+team members per regular Git workflow. [DVC Experiments] are directly woven into
+this commit history. But they also intend to avoid cluttering shared repos, so
```

Comment:
Sorry, I'm not sure I understand this

`DVC Experiments] are directly woven into this commit history.`

Let's try to write is simple and very explicit.

regular Git workflow, namely creating Git commits, pushing them to Github or from Github. 

It means that experiments become part of the Git history, they are visible in Git, etc. As we mentioned this is not idea since doesn't scale well, creates mess in Git history etc. DVC experiments still rely on Git and Github, but they are not directly attached ... so won't visible ... etc ..

Reply:
> Let's try to write is simple and very explicit.

Simplified in 0fb52cb3.

Reply:
I think the rest of your suggestions are great but do we want to elaborate so much on general motivation for DVC Experiments in this guide @shcheklein ? Maybe we should separately review that this is clearly included in the Overview page and/or the Running Exps page.

For now the only mention of motivation is "to avoid cluttering everyone's copies of the repo, by default experiments will only exist in the local environment where they were created."